lib/core/src/system/web/dom/html/css3d_renderer.rs

```diff
@@ -0,0 +1,269 @@
+//! This module contains the Css3dRenderer, a struct used to render CSS3D elements.
+
+use crate::prelude::*;
+
+use crate::display::object::DisplayObjectData;
+use crate::display::camera::Camera2d;
+use crate::display::camera::camera2d::Projection;
+use crate::system::web::dom::html::Css3dObject;
+use crate::system::gpu::data::JsBufferView;
+use crate::system::web::Result;
+use crate::system::web::create_element;
+use crate::system::web::dyn_into;
+use crate::system::web::NodeInserter;
+use crate::system::web::StyleSetter;
+use crate::system::web::dom::DomContainer;
+use crate::system::web::dom::ResizeCallback;
+use super::eps;
+
+use nalgebra::Vector2;
+use nalgebra::Matrix4;
+use wasm_bindgen::prelude::wasm_bindgen;
+use wasm_bindgen::JsValue;
+use web_sys::HtmlElement;
+use basegl_system_web::get_element_by_id;
+
+
+// ===================
+// === Js Bindings ===
+// ===================
+
+mod js {
+    use super::*;
+    #[wasm_bindgen(module = "/src/system/web/dom/html/snippets.js")]
```

Comment:
I would use inline js here. The js is super-small and having it inline would increase the understanding of what it really does.

Reply:
@wdanilo wait, but it's a 28 lines file.